app/graphql/types/group_type.rb

```diff
@@ -0,0 +1,10 @@
+module Types
+  class GroupType < Types::BaseObject
+    field :id, ID, null: false
+    field :name, String, null: false
+    field :description, String, null: false
+    field :group_users, [Types::GroupUserType], null: false
+    field :users, [Types::UserType], null: false
+    #field :checkins, [Types::CheckinType], null: false
+  end
```

Comment:
I am currently working on checkins and will let you know when I have it up. I ran into an unexpected issue.